specutils/io/registers.py

```diff
@@ -34,6 +34,16 @@ def decorator(func):
 
         logging.debug("Successfully loaded reader \"{}\".".format(label))
 
+        # Automatically register a SpectrumList reader for any data_loader that
+        # reads Spectrum1D objects. TODO: it's possible that this
+        # functionality should be opt-in rather than automatic.
+        if dtype == Spectrum1D:
```

Comment:
This should probably be an `isinstance` call, not a value equivalency check.

Reply:
`isinstance` applies to instances, not types themselves 😉

Reply:
Maybe it should be using `is` instead of `==` though.

Reply:
Haha, my brain turned off for a moment. I meant `is`, not `isinstance` -- my fingers took control. Anyway, it's the same premise as checking for `None` -- `is` checks for object identity, while `==` checks value.